docs/ecosystem_release/starrocks-kubernetes-operator.md

```diff
@@ -0,0 +1,163 @@
+# starrocks-kubernetes-operator
+
+## Notifications
+
+The Operator provided by StarRocks is used to deploy StarRocks clusters in the Kubernetes environment. The StarRocks cluster components include FE, BE, and CN.
+
+**User guide:** You can use the following methods to deploy StarRocks clusters on Kubernetes:
+
+- [Directly use StarRocks CRD to deploy StarRocks clusters](https://docs.starrocks.io/docs/deployment/sr_operator/)
+- [Deploying both the Operator and StarRocks clusters by using the Helm Chart](https://docs.starrocks.io/zh/docs/deployment/helm/)
+
+**Source codes:**
+
+- [starrocks-kubernetes-operator](https://github.com/StarRocks/starrocks-kubernetes-operator)
+- [kube-starrocks Helm Chart](https://github.com/StarRocks/starrocks-kubernetes-operator/tree/main/helm-charts/charts/kube-starrocks)
+
+**Download URL of the resources:**
+
+- **URL prefix**:
+
+    `https://github.com/StarRocks/starrocks-kubernetes-operator/releases/download/v${operator_version}/${resource_name}`
+
+- **Resource name**
+
+  - StarRocksCluster CRD: `starrocks.com_starrocksclusters.yaml`
+  - Default configuration file for StarRocks Operator: `operator.yaml`
+  - Helm Chart, including `kube-starrocks` Chart `kube-starrocks-${chart_version}.tgz`. The `kube-starrocks` Chart is divided into two subcharts: `starrocks` Chart `starrocks-${chart_version}.tgz` and `operator` Chart `operator-${chart_version}.tgz`.
+
+For example, the download URL for kube-starrocks chart v1.8.6 is: [kube-starrocks](https://github.com/StarRocks/starrocks-kubernetes-operator/releases/download/v1.8.6/kube-starrocks-1.8.6.tgz)
+
+**Version requirements**
+
+Kubernetes: 1.18 or later
+
+Go: 1.19 or later
+
+## **Release note**
+
+## 1.8
+
+**1.8.6**
+
+**Bug Fixes**
+
+Fixed the following issue:
+
+- An error  `sendfile() failed (32: Broken pipe) while sending request to upstream` is return during a Stream Load job. After Nginx sends the request body to FE, FE then redirects the request to BE. At this point, the data cached in Nginx may already be lost. [#303](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/303)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/db954e8c-7170-4bfe-99e5-754300bdb49b)
-> is returned